content/audio.md

```diff
@@ -129,6 +132,18 @@ This command will start PulseAudio after it's been stopped (this is not usually
 pulseaudio --start
 ```
 
+This command will check the status of PipeWire and show any errors if automatic restarts raised any errors:
+
+```
+systemctl --user status wireplumber
+```
+
+If you would like to monitor PipeWire, run:
```

Comment:
I'm not sure what this command is showing me, having just run it for the first time. A more specific description would be helpful.